posts/2021-05-06-Rust-1.52.0.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Announcing Rust 1.52.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.52.0. Rust is a
+programming language that is empowering everyone to build reliable and
+efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.52.0 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install]
+from the appropriate page on our website, and check out the
+[detailed release notes for 1.52.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1510-2021-03-25
+
+## What's in 1.52.0 stable
+
+This release is fairly small, primarily containing stabilizations of various APIs in the standard library.
```

Comment:
Since 1.52.0 has some fixes for the unsoundness for which CVE was issued (rust-lang/rust#81728, rust-lang/rust#83629), I guess it's worth mentioning it?

Reply:
I think a mention in the release notes (i.e. releases.md) isn't a bad idea, but none of those fixes are particularly note worthy, so I'd note call attention to them I think.

Reply:
Yeah, those are "normal" unsoundness bugs that happen to have a CVE attached to them. I don't think attaching a CVE to an issue should be a way to ensure issues are featured on the blog post.

Reply:
> I think a mention in the release notes (i.e. releases.md) isn't a bad idea, but none of those fixes are particularly note worthy

> those are "normal" unsoundness bugs that happen to have a CVE attached to them

Okay, thanks for clarifying!